App.js

```diff
@@ -1,26 +1,191 @@
-import React from "react"
-import styled from "styled-components/native"
+import React, { useEffect, useState } from "react"
+import styled from "styled-components"
+import { Image, View, ScrollView, Button, Linking, StyleSheet, Picker, Text, Vibration } from 'react-native'
+import moment from 'moment'
+import CustomButton from "./components/CustomButton"
+import Form from "./components/Form"
+
+
+
+
 
 const App = () => {
+
+  const [categories, setCategories] = useState('Climate')
+  const [news, setNews] = useState([])
+
+  const apiKey = `ceaaa691942f4f51925568bcf5183b80`
+  const url = `https://newsapi.org/v2/everything?q=${categories}&from=2020-01-01`
+
+
+  useEffect(() => {
+    fetch(url,
+      {
+        headers:
+          { Authorization: apiKey }
+      })
+      .then(res => res.json())
+      .then(json => {
+        setNews(json.articles)
+
+      })
+  }, [categories])
+
   return (
+
+
     <Container>
-      <Title>This is your cool app!</Title>
-      <Title>Go to App.js and start coding</Title>
-      <Title>💅💅💅</Title>
-    </Container>
+
+      <ScrollView>
+
+
+        <Image
+          source={require('./assets/bluesky.jpg')}
+          style={{ width: 350, height: 150 }} />
+        <View1>
+          <Title4>Welcome to Blue Sky News!</Title4>
+          <Title4>Updated from the world {moment().calendar()}</Title4>
+          <Picker
+            style={{ height: 50, width: 150, margin: 10, }}
```

Comment:
try to not use inline styling like this.
Stick to styled components or StyleSheet.